draft-normington-jsonpath.md

```diff
@@ -263,7 +270,8 @@ $['store']['book'][0]['title']
 for paths input to a JSONPath processor.
 \[1]
 Where a JSONPath processor uses JSONPath expressions as output paths,
-these will always be converted to the more general *bracket–notation*.
+these will always be converted to normalized JSONPath expressions
+which employ the more general *bracket–notation*.
 \[2]
 Bracket notation is more general than dot notation and can serve as a
 canonical form when a JSONPath processor uses JSONPath expressions as
```

Comment:
Do we want to avoid the use of "expressions" here so that it's not confused with query expressions (defined on line 290)?

Reply:
This is one of many places that uses "jsonpath expression".  We could all rename those into "query".  Holding off with that right now.